src/routes.js

```diff
@@ -301,5 +223,89 @@ export default [
     component: () => import(`./components/common/PageFeatureNotAvailable`),
     props: true
   },
-  { path: `*`, component: () => import(`./components/common/Page404`) }
+  {
+    path: `/:networkId`,
+    component: () => import(`./components/common/NetworkSetter`),
+    children: [
```

Comment:
should we make session screens children of network?

Reply:
Doesn't make too much sense right? They are they same no matter which network you are connected to

Reply:
my thinking is that it would simpler because the user wouldn't have to choose a network before a session flow. 

we could also share links to create / recovering / signing in / exploring for specific networks which may be valuable for support questions and for customers who want to share a link to creating an address on their own network.

... just a thought. doesn't have to be done here / now.